lisa/tools/ping.py

```diff
@@ -29,7 +37,7 @@ def ping_async(
         if package_size:
             args += f" -s {package_size}"
 
-        return self.run_async(args, force_run=True)
+        return self.run_async(args, force_run=True, sudo=sudo)
```

Comment:
keep arguments consistent between sync and async v3ersion. if the sudo must be True, remove it from async method, and set it always True.